specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2021-06-01-preview/examples/CheckNameAvailability.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "api-version": "2020-02-14-preview",
```

Comment:
api-versions need to be updated in these examples.

Reply:
Fix this and other places